common/components/BalanceSidebar/OfflineToggle.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  forceOfflineConfig as dForceOfflineConfig,
+  TForceOfflineConfig
+} from 'actions/config';
+import OfflineSymbol from 'components/ui/OfflineSymbol';
+import { connect } from 'react-redux';
+
+type sizeType = 'small' | 'medium' | 'large';
+
+interface OfflineToggleProps {
+  offline: boolean;
+  forceOffline: boolean;
+  forceOfflineConfig: TForceOfflineConfig;
+  size?: sizeType;
+}
+
+class OfflineToggle extends React.Component<OfflineToggleProps, {}> {
+  public render() {
+    const { forceOfflineConfig, offline, forceOffline, size } = this.props;
```

Comment:
If you want to avoid `forceOfflineConfig as dForceOfflineConfig` , you can either avoid destructuring to prevent name shadowing, or just destructure one level. E.g. `const {props} = this`